lib/db_connection/connection.ex

```diff
@@ -182,22 +187,23 @@ defmodule DBConnection.Connection do
         {:stop, {:shutdown, err}, s}
       {_, :after_connect} ->
         {timeout, backoff} = Backoff.backoff(backoff)
-        {:backoff, timeout, %{s | backoff: backoff}}
+        maybe_hibernate(hibernate?, {:backoff, timeout, %{s | backoff: backoff}})
```

Comment:
Should we wrap this one in hibernate?

Reply:
yep, this point is `disconnect`, so it not necessary to hibernate. Thanks